Makefile

```diff
@@ -35,6 +35,14 @@ endif
 wal-check: temp-install
 	$(prove_check)
 
+check: isolationcheck
+
+submake-isolation:
```

Comment:
It will be good to move `submake-isolation` target lower, closer to the `isolationcheck` target.